Tests/check_spider_dos.py

```diff
@@ -0,0 +1,8 @@
+# Tests potential DOS of SpiderImagePlugin when seeking in a non-stack file.
+# Run from anywhere that PIL is importable.
```

Comment:
This line can be deleted, because it needs to be run from somewhere it can open the test image.